src/containers/SearchPage/SelectNumberFromRangeFilter/SelectNumberFromRangeFilter.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { arrayOf, bool, func, node, object, string } from 'prop-types';
+
+import debounce from 'lodash/debounce';
+import classNames from 'classnames';
+import { Field } from 'react-final-form';
+
+import FilterPlain from '../FilterPlain/FilterPlain';
+import FilterPopup from '../FilterPopup/FilterPopup';
+import RangeInput from './RangeInput';
+
+import { FormattedMessage } from '../../../util/reactIntl';
+import css from './SelectNumberFromRangeFilter.module.css';
+
+const RADIX = 10;
+
+// Takes a single argument `valueRange`, expected to be a string that contains two numbers
+// separated by a comma, e.g: "0,1000". It extracts an object containing the keys `minValue` and `maxValue`
+// or null if the input is not valid. This function is used to convert query parameters into an object.
+const convertQueryParamToObject = valueRange => {
+  const [minValue, maxValue] = !!valueRange
+    ? valueRange.split(',').map(v => Number.parseInt(v, RADIX))
+    : [];
+  // We explicitly compare `minValue` to null as comparing to '0' returns false but '0' is a valid range value.
+  return !!valueRange && minValue != null && maxValue != null ? { minValue, maxValue } : null;
+};
+
+// Transforms a range object containing `minValue` and `maxValue`
+// into query parameter format, i.e. "0,1000"
+const formatToQueryParam = (range, queryParamName) => {
+  const { minValue, maxValue } = range || {};
+  // We explicitly compare `minValue` to null as comparing to '0' returns false but '0' is a valid range value.
+  const value = minValue != null && maxValue != null ? `${minValue},${maxValue}` : null;
+  return { [queryParamName]: value };
+};
+
+const SelectNumberFromRangeFilter = props => {
```

Comment:
I'd suggest that you rename this component and file.

This does not select a number from a range. It selects a range and the range needs to be an integer. 
(Btw. number in JavaScript is a double by default ([double precision floating point](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number#number_encoding)).

Suggestion: **IntegerRangeFilter**